Controller.php

```diff
@@ -280,4 +281,23 @@ private function getPrettyDateForSite($datetime, $period, $timezone)
 
         return $prettyDate;
     }
+
+    private function fetchSites($alert)
+    {
+        $sites = SitesManagerApi::getInstance()->getSitesWithAtLeastViewAccess();
```

Comment:
It doesn't place 50k in the select widget. It only shows eg 15 sites by default and only fetches 15 sites (unless configured differently with `site_selector_max_sites`)